packages/connect-wallet-modal/src/connectButtons/index.ts

```diff
@@ -1,4 +1,5 @@
-export { default as default } from './connectButton';
+// eslint-disable-next-line no-restricted-exports
+export { default } from './connectButton';
```

Comment:
Won't `export { default as default }` work here?

Reply:
Eslint gives another complain on it.
Indeed, there should be no such default export here, but it may break something if I just remove it now.